docs/source/algorithms/LoadNexus-v1.rst

```diff
@@ -12,9 +12,25 @@ Description
 The algorithm LoadNexus will read the given Nexus file and try to
 identify its type so that it can be read into a workspace. The file name
 can be an absolute or relative path and should have the extension .nxs
-or .nx5. Currently only Nexus Muon Version 1 files are recognised, but
-this will be extended as other types are supported such as
-:ref:`algm-LoadNexusProcessed`.
+or .nx5.
+
+The type of Nexus file is identified as follows:
+
+* If the file has a group of class ``SDS`` of name ``"definition"`` or ``"analysis"`` 
```

Comment:
Aha, I see. Interesting, `Mantid::NeXus::getNexusEntries()` must have been modified to take into consideration "analysis" for muon-v1 files, and that is documented. I'd go ahead and fix the "FIXME" comment ith the commented out variable, to prevent confusion.